conf.d/21_output_journald_gelf.conf

```diff
@@ -0,0 +1,23 @@
+# Mutate events to be suitable for pushing to gelf 
+filter {
+  if "journald" in [tags] {
```

Comment:
You need to add a new tag `gelf_output` or similarly called one, so that below you can check whether an event is tagged with `gelf_output` to be consumed by `output { gelf }`, otherwise same events will be picked up by other outputs as well.

Same tagging trick needs to be done for other outputs, i.e. cloudwatch.